dev/bots/service_worker_test.dart

```diff
@@ -147,7 +148,13 @@ Future<void> _rebuildApp({ required int version, required ServiceWorkerTestType
   );
   await runCommand(
     _flutter,
-    <String>['build', 'web', '--profile', '-t', target],
+    <String>[
+      'build', 'web', '--profile', '-t', target,
+       if(baseHref != null)
+       ...<String>['--base-href', baseHref],
+       if(output != null)
+       ...<String>['--output', output],
```

Comment:
```suggestion
       if(baseHref != null) ...<String>[
         '--base-href',
         baseHref,
       ],
       if(output != null) ...<String>[
         '--output',
         output,
       ],
```
Relevant style guide section: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#put-spread-inside-collection-if-or-collection-for-on-the-same-line